CUSTOMIZATION.md

```diff
@@ -87,9 +87,39 @@ The header may optionally define the following:
     this line.  Note: this value must be given as a string in double
     quotes, rather than as a number.
 
-## Home Page: Schedule, Syllabus, and Setup
+## Home Page: Schedule and Syllabus
 
 You should edit the sections titled `Schedule` and `Syllabus`
 so that they show what you're actually planning to teach and when.
-You should also delete irrelevant parts of the section titled `Setup`
+
+## Home Page: Setup
+
+You should irrelevant parts of the section titled `Setup`
 so that your learners don't try to install software that they won't need.
+After you edit the `Setup` section, you should edit the installation test script.
+
+`swc-installation-test-1.py` is pretty simple, and just checks that
+the students have a recent enough version of Python installed that
+will be able to parse `swc-installation-test-2.py`.
```

Comment:
"will be able to parse `swc-installation-test-2.py`" -> "they can run `swc-installation-test-2.py`"